src/lang/LanguageManager.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\lang;
+
+use function array_filter;
+use function array_map;
+use function explode;
+use function file_exists;
+use function is_dir;
+use function parse_ini_file;
+use function scandir;
+use function substr;
+use function is_array;
+use const INI_SCANNER_RAW;
+use const SCANDIR_SORT_NONE;
+
+class LanguageManager
+{
```

Comment:
Formatting: brace indentation should match the rest of PocketMine's classes

Reply:
This is not applicable you can fix it